sys-process/bashtop/bashtop-0.9.23.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
+
+DESCRIPTION="Resource monitor that shows usage and stats"
+HOMEPAGE="https://github.com/aristocratos/bashtop"
+SRC_URI="https://github.com/aristocratos/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+DOCS=( README.md )
+
+DEPEND="${RDEPEND}"
+RDEPEND=""
+
+src_prepare() {
+	default
+	sed -i -e 's/local//g' Makefile || die "Editing path failed"
+	sed -i -e '/DOCDIR/d' Makefile || die "Removing doc folder failed"
+}
+
+pkg_postinst() {
+	optfeature "CPU Temperature support" sys-apps/lm-sensors
+	optfeature "Update news and Theme Downlaod feature" net-misc/curl
```

Comment:
Typo. Just style issues I noticed, nothing big.

Reply:
Thanks, I will correct it.